oauth2_http/javatests/com/google/auth/oauth2/IdTokenCredentialsTest.java

```diff
@@ -0,0 +1,116 @@
+/*
+ * 2019 Google LLC
```

Comment:
This still needs the copyright
```suggestion
 * Copyright 2019 Google LLC
```

Reply:
done; could our formatter `mvn com.coveo:fmt-maven-plugin:format`  do all these checks?

Reply:
`maven-checkstyle-plugin` should verify the copyright headers if you run the tests.